code/src/java/gmgen/util/MiscUtilities.java

```diff
@@ -24,19 +22,13 @@
 {
 	private MiscUtilities()
 	{
-		//Do not allow instantiation of utility class
```

Comment:
Can't be pmd cause we fatal on errors; can't be checkstyle cause codacy would have flagged it. Maybe its old?